stylesheets/_content.less

```diff
@@ -315,6 +315,7 @@
     width: 70%;
     max-width: 880px;
     margin-left: 30%;
+    padding-right: 4%;
```

Comment:
Since the layout is otherwise done with margin I'd suggest using that here, then just remove the width property to allow it to be however wide it needs to be.